debugview/build.gradle

```diff
@@ -0,0 +1,49 @@
+plugins {
+    alias libs.plugins.android.library
+    alias libs.plugins.kotlin.android
+}
+
+// TODO Uncomment apply plugin once debugview package is ready to be published.
+//if (!project.getProperties()["ANDROID_VARIANT_TO_PUBLISH"].contains("customEntitlementComputation")) {
+//    apply plugin: "com.vanniktech.maven.publish"
+//}
+
+apply from: "$rootProject.projectDir/library.gradle"
+
+android {
+    namespace 'com.revenuecat.purchases.debugview'
```

Comment:
Nice

Reply:
Should this be `com.revenuecat.purchases.ui` maybe? Are you thinking of having a separate one for "future" UI? 👀

Reply:
Hmm maybe... I was thinking that for other ui projects, those would belong to other modules/dependencies, but maybe we could scope all those under `ui`. So maybe rename to `com.revenuecat.purchases.ui.debugview`. Wdyt?

Reply:
I will do this rename in a followup PR to avoid conflicts